coordinator/cql/src/main/java/org/apache/cassandra/stargate/transport/internal/CqlServer.java

```diff
@@ -269,6 +244,13 @@ private InetSocketAddress getSocket() {
 
   public static class ConnectionTracker implements Connection.Tracker {
     // TODO: should we be using the GlobalEventExecutor or defining our own?
+    private static final ChannelMatcher PRE_V5_CHANNEL =
```

Comment:
can we add a `_MATCHER` extension here, was confused when I was reading, was thinking this is no a function